rag_apps/contextual_ai_rag/main.py

```
@@ -0,0 +1,306 @@
+            api_key=nebius_api_key
+        )
+        
+        enhancement_prompt = f"""Based on the original query and AI response below, provide a brief enhancement that adds key insights, improves clarity, or suggests relevant follow-up questions. Keep it concise and valuable.
+
+Original Query: {query}
+AI Response: {original_response}
+
+Enhancement:"""
+        
+        enhanced = nebius_llm.complete(enhancement_prompt)
+        return f"{original_response}\n\n**💡 Enhanced Insights:**\n{enhanced}"
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)